hooks/useAddressValidation.ts

```diff
@@ -0,0 +1,28 @@
+import { BridgeChain } from "@/types";
+import isCENNZAddress from "@/utils/isCENNZAddress";
+import isEthereumAddress from "@/utils/isEthereumAddress";
+import { MutableRefObject, useEffect, useRef } from "react";
+
+interface AddressValidationHook {
+	inputRef: MutableRefObject<HTMLInputElement>;
+}
+
+export default function useAddressValidation(
+	address: string,
+	addressType: BridgeChain
+): AddressValidationHook {
+	const inputRef = useRef<HTMLInputElement>();
```

Comment:
keen for a run-through for using inputRef at some point

Reply:
sure thing, which one in particular tho? The usage of `useRef`?

Reply:
yup!